docs/design/latency_definitions/index.md

```diff
@@ -0,0 +1,141 @@
+# Latency definition
+
+CARET mainly measures the following
+
+- Callback latency
+- Communication latency
+- Node latency
+- Path latency
+
+The simplified sequence diagram shown below illustrates each definition.
+
+![Latency overview](../../imgs/latency_overview.png)
+
+Here, the horizontal axis represents time and the vertical axis represents layers.
+The red line represents message flow.
+A message is received in the subscription callback and the processed data is published to the next node.
+In this way, information is propagated from the sensor node to the actuator node.
+
+CARET measures the following three main locations for latency calculation.
```

Comment:
```suggestion
CARET samples events for latency calculation. The following three items are main types of the events.
```

Reply:
addressed in: [b104708](https://github.com/tier4/CARET_doc/pull/106/commits/b104708b0378c76f80d2ba457892ff9ad0dbfdc4)